src/views/homepage/components/featured-content-grid/search-featured-card.module.css

```diff
@@ -2,9 +2,8 @@
 	background: url('../../img/search-card-graphic.svg');
 	background-color: var(--token-color-foreground-strong);
 	background-size: cover;
-	background-position: center left;
+	background-position: center top;
```

Comment:
🗒️ Kelly & I swapped out the graphic for this, and it needed to be re-positioned.